rollup.config.js

```diff
@@ -138,6 +140,27 @@ async function main()
         const input = path.join(basePath, 'src/index.ts');
         const freeze = false;
 
+        const tempPlugins = [...plugins];
+        const tempProdPlugins = [...prodPlugins];
+        const tempBundlePlugins = [...prodBundlePlugins];
+
+        if (transpile === 'es6')
+        {
+            const ts = typescript({
+                tsconfig: path.resolve(__dirname, 'tsconfig.node.json'),
+            });
+
+            tempPlugins.splice(5, 1);
+            tempPlugins.splice(5, 0, ts);
+            tempPlugins.pop();
+            tempProdPlugins.splice(5, 1);
+            tempProdPlugins.splice(5, 0, ts);
+            tempProdPlugins.splice(tempProdPlugins.length - 2, 1);
+            tempBundlePlugins.splice(6, 1);
+            tempBundlePlugins.splice(6, 0, ts);
+            tempBundlePlugins.splice(tempBundlePlugins.length - 2, 1);
```

Comment:
🤔 

I think there are two important pieces of adding a good hack:

* It can be easily removed
* It's clear why it's being added

This change creates a bunch of temp variables making it annoying to extract, and it is opaque as to what's happening here because of all the popping and splicing.

A better and more transparent approach would be to copy-paste the plugin configuration from above and override the variables. And there should be a comment here that this is only for the Node package and can be removed in v7. 

Also, for this hack we do not need bundle plugins, as we don't generate the bundle for `@pixi/node`.